spec/controllers/evaluation_controller_spec.rb

```diff
@@ -150,5 +150,36 @@
     end
 
   end
+  
+  describe "POST #upload" do
+    it "accepts .xlsx files for uploading" do
+      @file = fixture_file_upload('/StatisticsReport.xlsx', 'application/vnd.ms-excel')
+      post :upload, :data_file => @file
+      expect(response).to redirect_to("/evaluation")
+      expect(response.body).to have_content("Term")
+    end
+    it "displays an error message when an incorrect filetype is uploaded" do
+      @file = fixture_file_upload('/ruby-capybara.zip', 'application/zip')
+      post :upload, :data_file => @file
+      expect(response).to have_http_status(200)
+      #TODO: fix finding content
+      expect(response.body).to have_content("Error with file upload, please check your file")
+    end
+    it "parses file and extracts data into evaluation model" do
+      @file = fixture_file_upload('/StatisticsReport.xlsx', 'application/vnd.ms-excel')
+      post :upload, :data_file => @file
+      #TODO: change expect
```

Comment:
Not sure what this TODO means. Please either clarify or do what it is signifying.

Same below.